packages/terra-form-select/src/_Tag.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './_Tag.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The content of the tag.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Callback function triggered when the tag is deselected.
+   */
+  onDeselect: PropTypes.func.isRequired,
+  /**
+   * The value of the tag.
+   */
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+};
+
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
We should still add an aria label to the span element that's interactionable.

Reply:
I'm fine with that. I found a defect on it's repo that says it won't go away unless you update.